templates/layout/page.html.twig

```diff
@@ -56,23 +56,10 @@
  * @see html.html.twig
  */
 #}
-
-{#
-{{ kint() }}
-#}
-
-{% if layout_option == 'full' or 'fluid' %}
```

Comment:
I think we still need this, unless it's being set somewhere else (which it likely should be - preprocess page would be cleaner). The 'full-width' body selector is used to adjust the layout container boundaries iirc.

Reply:
The variable in there isn't being used in `page.html.twig` only in `html.html.twig`

Reply:
D'oh, you're right. I was looking at one template and thinking of the other. Ok, this all looks great.